cfgov/apache/conf.d/headers.conf

```diff
@@ -5,11 +5,16 @@ LimitRequestFieldSize 8190
 KeepAlive On
 MaxKeepAliveRequests 500
 
-
 Header always set X-Frame-Options SAMEORIGIN
 Header always set X-XSS-Protection: "1; mode=block"
 Header always set X-Content-Type-Options: nosniff
 
+# Add LimitRequestBody if the value is provided in the LIMIT_REQUEST_BODY 
+# variable.
```

Comment:
Trivial suggestion to get the comment on one line and be consistent with the one below.

```suggestion
# Add LimitRequestBody when LIMIT_REQUEST_BODY env var is set
```